openid-4-verifiable-presentations-1_0.md

```diff
@@ -417,7 +417,7 @@ This specification defines the following values for the `client_id_scheme` param
 
 * `pre-registered`: This value represents the [@!RFC6749] default behavior, i.e., the Client Identifier needs to be known to the Wallet in advance of the Authorization Request. The Verifier metadata is obtained using [@!RFC7591] or through out-of-band mechanisms.
 
-* `redirect_uri`: This value indicates that the Verifier's redirect URI is also the value of the Client Identifier. In this case, the Authorization Request MUST NOT be signed, the Verifier MAY omit the `redirect_uri` Authorization Request parameter, and all Verifier metadata parameters MUST be passed using the `client_metadata` or `client_metadata_uri` parameter defined in (#vp_token_request). 
+* `redirect_uri`: This value indicates that the Verifier's redirect URI is also the value of the Client Identifier. In this case, the Authorization Request MUST NOT be signed. The Verifier MAY omit the `redirect_uri` Authorization Request parameter. All Verifier metadata parameters MUST be passed using the `client_metadata` or `client_metadata_uri` parameter defined in (#vp_token_request). When Response Mode `direct_post` is used, these clauses apply to the `response_uri` value instead of `redirect_uri` value.
```

Comment:
this statement is not clear

it is not clear why the client uid must (or is) the same of the redirect uri.
it is not clear why the authorization must not be signed.

Reply:
"it is not clear why the client uid must (or is) the same of the redirect uri."

there is no way to link the client_id to a redirect_uri in this scheme then making both the same

"it is not clear why the authorization must not be signed."

I think we can drop that requirement. It was used in the past (before client id schemes were introduced) to differentiate redirect_uri as client id and other options. With the client id scheme `redirect_uri`, it is no longer needed.